spec/requests/admin/base_items_requests_spec.rb

```diff
@@ -18,59 +18,64 @@
     end
   end
 
-  # TODO aren't organization_admins not allowed to view base items?
-  # also, some of these tests are sending organization.id instead of BaseItem.id as args
   context "When logged in as an organization admin" do
     before do
       sign_in(organization_admin)
     end
 
     describe "GET #new" do
-      it "returns http success" do
+      it "denies access and redirects" do
```

Comment:
I don't think you even need these specs - every route under `admin` is blocked to every user besides admins. We don't need specs for every one of those routes. Maybe pick one representative one (index?) and leave it at that.

Reply:
addressed in commit [a861839](https://github.com/rubyforgood/human-essentials/pull/4665/commits/a861839e9bd172aca6e26dbf1329d2d6815b6dec)